src/docker-osx-dev

```diff
@@ -735,12 +735,18 @@ function check_homebrew {
 function install_dependencies {
   brew_update
 
-  brew_install "caskroom/cask/brew-cask" "Cask" "brew cask" false
-  brew_install "boot2docker" "Boot2Docker" "boot2docker" false
-  brew_install "docker-compose" "Docker Compose" "docker-compose" false
-  brew_install "docker-machine" "Docker Machine" "docker-machine" false
-  brew_install "fswatch" "fswatch" "fswatch" false
-  brew_install "coreutils" "GNU core utilities" "greadlink" false
+  log_info "Setting up Cask"
+  if ! brew tap Caskroom/cask; then
+    log_error "Failed to set up Cask. Check the log output above for reasons."
+    exit 1
+  fi
+
+  brew_install "caskroom/cask/brew-cask" "Cask" "brew cask list" false
```

Comment:
All of these set the `use_cask` flag to false. It can probably be removed entirely from these calls and the `brew_install` function itself.